### DISCUSSION TITLE
⚠ Academic Reverification Required

### DISCUSSION THREAD

**Discussion by DamianLammenda** [community/community#59777] [Other Feature Feedback, Questions, & Ideas]
Question

I have a problem when it comes to revalidate my studies, when I choose the institution where I am studying, it asks me for an e-mail with the domain of that school, and the school does not provide e-mails, so it is impossible for me to continue.
![image](https://github.com/community/community/assets/99482368/87315c21-98b1-4b45-9f8c-b3b9d4995581)
before it did not ask for it
Can anyone help me, thank you.

**Comment by HarlanHaller** [comment]
Your best bet would be to either contact someone at Coderhouse or GitHub support
